docs/src/content/en/docs/frameworks/agentic-uis/copilotkit.mdx

```diff
@@ -11,139 +11,139 @@ import Image from "next/image";
 CopilotKit provides React components to quickly integrate customizable AI copilots into your application.
```

Comment:
Do we link to CopilotKit docs / package / GitHub so users can quickly get to the source?